learning_resources/serializers.py

```diff
@@ -0,0 +1,127 @@
+"""Serializers for learning_resources"""
+from rest_framework import serializers
+
+from learning_resources import models
+
+COMMON_IGNORED_FIELDS = ("created_on", "updated_on")
+
+
+class LearningResourceInstructorSerializer(serializers.ModelSerializer):
+    """
+    Serializer for LearningResourceInstructor model
+    """
+
+    class Meta:
+        model = models.LearningResourceInstructor
+        fields = "__all__"
+
+
+class LearningResourceTopicSerializer(serializers.ModelSerializer):
+    """
+    Serializer for LearningResourceTopic model
+    """
+
+    class Meta:
+        model = models.LearningResourceTopic
+        fields = ["id", "name"]
+
+
+class LearningResourceOfferorField(serializers.Field):
+    """Serializer for LearningResourceOfferor"""
+
+    def to_representation(self, value):
+        """Serializes offered_by as a list of OfferedBy names"""
+        return [offeror.name for offeror in value.all()]
+
+
+class LearningResourceContentTagField(serializers.Field):
+    """Serializer for LearningResourceContentTag"""
+
+    def to_representation(self, value):
+        """Serializes resource_content_tags as a list of OfferedBy names"""
+        return [tag.name for tag in value.all()]
```

Comment:
Custom serializers are one place where drf-spectacular will want us to add a simple [`extend_schema_field`](https://drf-spectacular.readthedocs.io/en/latest/customization.html#step-3-extend-schema-field-and-type-hints).

*Just a commnet. I don't think we should do anything about it until this API gets hooked up to spectacular.*